bridges/evm/integration_examples/harmony/truffle-config.js

```diff
@@ -0,0 +1,103 @@
+// * Use this file to configure your truffle project. It's seeded with some
+// * common settings for different networks and features like migrations,
+// * compilation and testing. Uncomment the ones you need or modify
+// * them to suit your project as necessary.
+// *
+// * More information about configuration can be found at:
+// *
+// * truffleframework.com/docs/advanced/configuration
+// *
+// * To deploy via Infura you'll need a wallet provider (like truffle-hdwallet-provider)
+// * to sign your transactions before they're sent to a remote public node. Infura accounts
+// * are available for free at: infura.io/register.
+// *
+// * You'll also need a mnemonic - the twelve word phrase the wallet uses to generate
+// * public/private key pairs. If you're publishing your code to GitHub make sure you load this
+// * phrase from a file you've .gitignored so it doesn't accidentally become public.
+// *
+// */
+require("dotenv").config();
+const { TruffleProvider } = require("@harmony-js/core");
+//Local
+const local_mnemonic = process.env.LOCAL_MNEMONIC;
+const local_private_key = process.env.LOCAL_PRIVATE_KEY;
+const local_url = process.env.LOCAL_0_URL;
+//Testnet
+const testnet_mnemonic = process.env.TESTNET_MNEMONIC;
+const testnet_private_key = process.env.TESTNET_PRIVATE_KEY;
```

Comment:
Why we need both mnemonic and private key? `TruffleProvider`cannot generate the private key from mnemonic?

Reply:
Fix by adding reference in the coments